apps/mdn/mdn-aws/k8s/percona-toolkit-secrets.yaml.example

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: percona-toolkit-ptkill-secrets
+  namespace: percona-toolkit
+type: Opaque
+data: {}
+stringData:
+    pt_cmd: pt-kill --log --busy-time=1h --kill h=somehost,D=somedb,u=someuser,p=SuperSecurePassword
```

Comment:
It is hard for me to imagine a legitimate query that takes more than 1 hour. The logging includes the SQL, so we can investigate what queries are getting stuck 👍